frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/EditFieldDrawer.tsx

```diff
@@ -60,8 +62,10 @@ export const EditFieldDrawer = (): JSX.Element | null => {
 
   const basicFieldText = useMemo(() => {
     if (!fieldToEdit?.fieldType) return ''
+    if (isMyInfo(fieldToEdit))
+      return MYINFO_FIELD_CONSTANTS[fieldToEdit.myInfo.attr].value
```

Comment:
nit: wrap with braces since its multiline for cleanliness.

Reply:
sounds good, will do!